usr/share/device-quirks/scripts/aokzoe/aokzoea1ar07.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+### AOKZOE A1 Quirks
+
+if [ $(whoami) != 'root' ]; then
+	echo "You must be root to run this script."
+   exit 1
+fi
+ 
+### Unlock Filesystem
+
+frzr-unlock
```

Comment:
this shouldn't be necessary if we install all the bin files to the firmware folder using PKGBUILD

Reply:
True, I'll update that.

Reply:
Updated to have EDID bin already placed in appropriate location and removed frzr-unlock.